From: Bastien Montagne <montagne29@wanadoo.fr>
Date: Thu, 2 Jul 2015 22:32:11 +0200
Subject: blender_desktop

- Linux  XDesktop  spec:  add  '%f'  file  specifier  to  'Exec'  field.
- Also  reorder  alphabetically  generic  names  &  comments,  and  add
  french  comment.
- Add  a  'Keyword'  field.

Signed-off-by: Matteo F. Vescovi <mfv@debian.org>
---
 release/freedesktop/blender.desktop | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/release/freedesktop/blender.desktop b/release/freedesktop/blender.desktop
index 1620815..a28a815 100644
--- a/release/freedesktop/blender.desktop
+++ b/release/freedesktop/blender.desktop
@@ -1,14 +1,16 @@
 [Desktop Entry]
 Name=Blender
-GenericName=3D modeller
-GenericName[es]=modelador 3D
+GenericName=3D modeler
 GenericName[de]=3D-Modellierer
-GenericName[fr]=modeleur 3D
+GenericName[es]=Modelador 3D
+GenericName[fr]=Modeleur 3D
 GenericName[ru]=Редактор 3D-моделей
 Comment=3D modeling, animation, rendering and post-production
-Comment[es]=modelado 3D, animación, renderizado y post-producción
 Comment[de]=3D-Modellierung, Animation, Rendering und Nachbearbeitung
-Exec=blender
+Comment[es]=Modelado 3D, animación, renderizado y post-producción
+Comment[fr]=Modelage 3D, animation, rendu et post-production
+Keywords=3d;cg;modeling;animation;painting;sculpting;texturing;video editing;video tracking;rendering;render engine;cycles;game engine;python;
+Exec=blender %f
 Icon=blender
 Terminal=false
 Type=Application
